Code review request.

SunnySparks opened the pull request:
This pull request is open in order to get the code reviewed.

SunnySparks committed:
Add icons

Icons are added to the repo

SunnySparks committed:
fix overlapping problem

·Change the flex positioning so overlapping between the video and recommendations thumbnails won't occur.
·Replace emojis for proper icons on video recommendations.

SunnySparks committed:
Align reaction icons in the title

Create the subscribe button and aligne the share, like and dislike buttons to a more correct scale.

SunnySparks committed:
Final commit

·Replace share button for an icon, so resizing would not cause conflicts.
·Resize seachbar to make it look as wideas the one on the original website.
·New classes are added (row, row-wrap, and column) to replace the repeated properties of flex-direction: row, flex-direction: column, and flex-flow: row wrap. Only the elements that caused conflicts when assigned the new classes are kept untouched.

elmejdki reviewed (changes requested):
# Greate job so far team :clap: 
## status: require changes

[Mandatory] Guys never close a pull request if it's not approved cause that way we will not be able to see all your code in the **PR** and add our comments, so what are you going to do now, please create a new repository and put your changes here don't remove this one so we can reference it for commit messages.

#### some notes that I have seen:
- your page is not showing out correct your have sections overlapping over each other, so you need to work on that, I think that you have a big resolution screen so make always in mind that most of the users are having laptops with lower screen resolution like 1027x678, try to zoom on like around 150% to see what I'm talking about.
![image](https://user-images.githubusercontent.com/34653764/84214712-14566000-aabc-11ea-9566-9b9954bd9e1e.png)

- your live version link is not working you can use [githack](https://raw.githack.com/)

### please make sure to do the requirements before opening another pull request.
_if you needed help with anything don't hesitate to contact me on slack, you can find me as **Zakariae El Mejdki**._

Alexoid1 committed:
Responsive error fixed giving tho the cointainer a min with and setting a fix width and heigth to the img

SunnySparks committed:
Delete outdated screenshot

This screenshot will be replaced for a more recent one that matches the changes on the website

SunnySparks committed:
Add updated screenshot

This screenshot will replace the old one, in order to show the changes on the website.

SunnySparks committed:
Update README.md

Add the updated live version of the website
